### DISCUSSION TITLE
Documentation finding report

### DISCUSSION THREAD

**Discussion by paulesco** [apache/devlake#8107] [Q&A] [answered]
Hello devlake community,

I found a little error in the documentation and I was wondering how could I report or contribute to change it. 

It is a little mistake, the domain layer schema diagram shows that refs.repo_id is an int, but it is a varchar, as the DW Detail says.
https://devlake.apache.org/docs/v1.0/DataModels/DevLakeDomainLayerSchema

Best,

Paul

[a comment by an automated account is not shown]

**Comment by klesh** [comment]
@paulesco Hi, thanks for reporting the issue.
The answer from @dosubot is pretty solid, you may following the guide and submit your PR on https://github.com/apache/incubator-devlake-website

**Reply by paulesco** [reply]
Hello @klesh, thanks for your guidelines. I just created the PR, not exactly as the bot suggested, because I wanted to change the domain-layer-schema-diagram.svg image, which had the mistake, not the .md file.

**Reply by klesh** [reply]
@paulesco Great, thanks for your contribution.
